styles.css

```diff
@@ -0,0 +1,1917 @@
+  margin-bottom: 3.2vw;
+}
+
+li.idioma img {
+  margin-left: 3.2vw;
+  width: 12.8vw;
+}
+
+.contacto {
+  background-color: #fff;
+}
+
+.fondo {
+  border-top-left-radius: 21.3vw;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right 6.4vw;
+  align-items: center;
+  background-color: #6070ff;
+  background-image: url(media/mobile1.png);
+}
+
+.contacto h2 {
+  color: #fff;
+  font-size: 40px;
+  margin-top: 27.467vw;
+  line-height: 52px;
+  font-weight: 700;
+}
+
+.contacto input {
+  width: 87.2vw;
+  height: 12.8vw;
+  background: #fff;
+  border: 0.267vw solid #cfd8dc;
+  border-radius: 2.13vw;
+  font-size: 4.53vw;
+  margin-bottom: 5.3vw;
+}
+
+button:hover {
+  color: #fff;
+  background-color: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+button:active {
+  color: #fff;
+  background: #2230d2;
+}
+
+button:disabled {
+  background-color: transparent;
+  color: #5e6c84;
+  border-color: #c1c7d0;
+}
+
+#nohay {
+  display: block;
+}
+
+.mobile-only {
+  display: block; /* Mostrar en la versión móvil */
+}
+
+.desktop-only {
+  display: none; /* Ocultar en la versión de escritorio */
+}
+
+@media (min-width: 768px) {
+  .escritorio {
+    visibility: visible;
+    position: static;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 26.6px;
+    gap: 24px;
+    color: #344563;
+  }
+
+  .media {
+    margin: 0 10.764vw 0;
+  }
+
+  .mobile-only {
+    display: none; /* Ocultar en la versión de escritorio */
+  }
+
+  .desktop-only {
+    display: block; /* Mostrar en la versión de escritorio */
+    margin-left: -10px;
+  }
+
+  h2 {
+    font-size: 1.25vw;
+    margin: 0;
+    padding-top: 1.806vw;
+    padding-bottom: 1.806vw;
+  }
+
+  #menu-boton {
+    visibility: hidden;
+    position: absolute;
+  }
+
+  .enter {
+    background-image: url(media/escritorio1.png);
+    background-position: center;
+    border-bottom-left-radius: 10vw;
+  }
+
+  .enter-margen {
+    margin-left: 18.056vw;
+  }
+
+  .h1-media {
+    font-size: 3.3vw;
+    line-height: 4.167vw;
+    letter-spacing: 0.026vw;
+    padding-top: 20vw;
+    max-width: none;
+    margin-left: 18.056vw;
+  }
+
+  div.enter .info {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.389vw;
+    line-height: 1.94vw;
+    max-width: 74ch;
+    margin-top: 0.83vw;
+    margin-bottom: 0.83vw;
+  }
+
+  .clase p {
+    font-size: 0.83vw;
+    padding: 0.278vw 0.83vw;
+  }
+
+  .contacto p {
+    font-size: 1.389vw;
+    line-height: 1.94vw;
+    width: 39.514vw;
+    margin: 0 0 1.66vw 0;
+  }
+
+  div.conectar h4 {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.1vw;
+    margin-top: 0;
+    margin-bottom: 0.972vw;
+  }
+
+  .iconos {
+    margin-left: 5px;
+    gap: 1.389vw;
+    margin-top: 20px;
+  }
+
+  .enter .conectar {
+    margin-bottom: 17.708vw;
+  }
+
+  .tarjetas {
+    margin: 6.861vw;
+  }
+
+  .trabajo {
+    border-radius: 1.667vw;
+  }
```

Comment:
- [ ] Kudos for the UI, you almost approved. I suggest adding the hover animations to the cards as you can see in the Figma design to be similar to it, for more details check the image below it has a blue border and a shadow below the card. 👇🏽 

![image](https://github.com/maximoortelli/Portfolio-desktop-version/assets/58642949/b021d107-ac12-4a4a-91a2-c0f9b5c146db)

Reply:
Done, thank you franclobo 👍😃